pkg/trace/stats/sublayers_test.go

```diff
@@ -226,155 +226,30 @@ func TestComputeSublayers(t *testing.T) {
 			},
 			[]SublayerValue{
 				sublayerValueService("alert", 35),
-				sublayerValueService("pg", 12),
+				sublayerValueService("pg", 13),
 				sublayerValueService("pg-read", 15),
-				sublayerValueService("redis", 27),
+				sublayerValueService("redis", 28),
 				sublayerValueService("render", 15),
 				sublayerValueService("rpc1", 30),
 				sublayerValueService("web-server", 15),
-				sublayerValueType("cache", 27),
-				sublayerValueType("db", 27),
+				sublayerValueType("cache", 28),
+				sublayerValueType("db", 28),
 				sublayerValueType("rpc", 65),
 				sublayerValueType("web", 30),
 				sublayerValueCount(7),
 			},
 		},
 	}
+	c := NewSublayerCalculator()
 
 	for _, test := range tests {
-		values := ComputeSublayers(test.trace)
+		values := c.ComputeSublayers(test.trace)
 		sort.Sort(sublayerValues(values))
 
 		assert.Equal(test.values, values, "test: "+test.name)
 	}
 }
 
-func TestBuildTraceTimestamps(t *testing.T) {
```

Comment:
We don't care about whatever these tests were testing anymore?

Reply:
Now, the building of timestamps is integrated in `computeExecDuration`.
I could test the content of `sublayerCalculator` after calling `computeExecDuration` but I think that's a bit weird.
If I want to test the building of timestamp, I should split it in its own function.

But because we only use the timestamps in `computeExecDurations`, if `computeExecDurations` works well, we are good.
What do you think?